### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/tutorial/static-files.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/tutorial/static-files.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/tutorial/static-files.md)

Discussion: #9193

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2fe95dfdc2c01b450dc2d43f21292bf929b57d87 at: https://ad9906ce.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit df50159a0be59e8bf6dd6add2cfde510832fb21b at: https://5ebb8960.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

File: docs/tr/docs/tutorial/static-files.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Bir `StaticFiles()` örneğini belirli bir yola bağlayın.
+
+```Python hl_lines="2  6"
+{!../../../docs_src/static_files/tutorial001.py!}
+```
+
+!!! note "Teknik Detaylar"
+    Projenize dahil etmek için `from starlette.staticfiles import StaticFiles` kullanabilirsiniz.
+
+    **FastAPI**, geliştirici olarak size bir kolaylık sağlamak amacıyla `starlette.staticfiles`'ı `fastapi.staticfiles` olarak sağlar. Ancak `StaticFiles` sınıfı aslında doğrudan Starlette'den gelir.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/static-files.md:17]:
```suggestion
    **FastAPI**, geliştiricilere kolaylık sağlamak amacıyla `starlette.staticfiles`'ı `fastapi.staticfiles` olarak sağlar. Ancak `StaticFiles` sınıfı aslında doğrudan Starlette'den gelir.
```

File: docs/tr/docs/tutorial/static-files.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Bağlama (Mounting) Nedir?
+
+"Bağlamak", belirli bir yola tamamen "bağımsız" bir uygulama eklemek anlamına gelir ve ardından tüm alt yolları işler.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/static-files.md:21]: Burası kontrol edilecek.

File: docs/tr/docs/tutorial/static-files.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Bu, bir `APIRouter` kullanmaktan farklıdır çünkü bağlanmış bir uygulama tamamen bağımsızdır. Ana uygulamanızın OpenAPI ve dokümanlar, bağlanmış uygulamadan hiçbir şey içermez, vb.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/static-files.md:23]:
```suggestion
Bu, bir `APIRouter` kullanmaktan farklıdır çünkü bağlanmış bir uygulama tamamen bağımsızdır. Ana uygulamanızın OpenAPI ve dokümanları, bağlanmış uygulamadan hiçbir şey içermez, vb.
```

File: docs/tr/docs/tutorial/static-files.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+[Advanced User Guide](../advanced/index.md){.internal-link target=_blank} bölümünde daha fazla bilgi edinebilirsiniz.
+
+## Detaylar
+
+`"/static"` ifadesi, bu "alt uygulamanın" "bağlanacağı" alt yolu belirtir. Bu nedenle, `"/static"` ile başlayan her yol, bu uygulama tarafından ele alınır.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/static-files.md:29]: Burası kontrol edilecek.

**Review by hasansezertasan — COMMENTED** [review]
Not ready for merging.

**Comment by tiangolo** [comment]
📝 Docs preview for commit a89b92bfb92c1a7ada6debd7de7765a8a4b7bb32 at: https://72f0fd03.fastapitiangolo.pages.dev

**Comment by hasansezertasan** [comment]
@alejsdev, it's ready to get merged.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 93b97361554d27ffe4ebfc2124469dceeceff3d7 at: https://431a1362.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1de27ef507ba20c42a3e074a0687d111515e0975 at: https://b1fb2ef0.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 166f2107eecf3535b077187c12065bf78287236c at: https://969829b3.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Nice, thanks! :cat: :v: @hasansezertasan @OzgunCaglarArslan

**Pull request merged**